docs/contributing/howto/index.md

```diff
@@ -0,0 +1,7 @@
+# How-to Guides
+
+These are the developer guides for how to do specific things with the project.
+
+1. [Add model and API endpoints](add-model-and-api-endpoints.md)
+1. [Create initial data migrations](create-initial-data-migrations.md)
```

Comment:
Line 6: Change to 'Create initial data scripts' - and maybe change the name of the linked .md file too?

Reply:
I don't think we should change the wording here.

Migrations is the term Django uses for this type of scripts. They don't work alone, but have to be run by Django's migration system. https://docs.djangoproject.com/en/5.1/topics/migrations/

It took a while to compare and figure out the best format to keep the initial data and insert them into the database. I should write up a DR to document why.